embedmd/command.go

```diff
@@ -68,20 +68,38 @@ func fields(s string) ([]string, error) {
 	var args []string
 
 	for s = strings.TrimSpace(s); len(s) > 0; s = strings.TrimSpace(s) {
+		var sep int
 		if s[0] == '/' {
-			sep := strings.IndexByte(s[1:], '/')
+			sep = nextSlash(s, 1)
 			if sep < 0 {
 				return nil, errors.New("unbalanced /")
 			}
-			args, s = append(args, s[:sep+2]), s[sep+2:]
 		} else {
-			sep := strings.IndexByte(s[1:], ' ')
+			sep = strings.IndexByte(s[1:], ' ')
 			if sep < 0 {
 				return append(args, s), nil
 			}
-			args, s = append(args, s[:sep+1]), s[sep+1:]
 		}
+		args, s = append(args, s[:sep+1]), s[sep+1:]
 	}
 
 	return args, nil
 }
+
+// nextSlash will find the index of the next unescaped slash in a string.
+func nextSlash(s string, offset int) int {
+	sep := strings.IndexByte(s[offset:], '/')
+	if sep < 0 {
+		return sep
+	}
+
+	// build up an offset for the original string.
+	sep += offset
```

Comment:
Explanation: This is so that recursed calls will bubble up an offset against the original string, rather than for the subslice they actually get an index for.